ci/setup-build-test-gdk.ps1

```diff
@@ -57,20 +57,23 @@ if (Test-Path env:BUILD_ALL_CONFIGURATIONS) {
     $test_repo_map = "EmptyGym"
     $test_project_name = "GDKTestGyms"
 
-    $tests += [TestSuite]::new("$test_repo_url", "$test_repo_branch", "$test_repo_relative_uproject_path", "$test_repo_map", "$test_project_name", "TestResults", "SpatialGDK", "bEnableUnrealLoadBalancer=false$user_gdk_settings", $True)
+    $tests += [TestSuite]::new("$test_repo_url", "$test_repo_branch", "$test_repo_relative_uproject_path", "$test_repo_map", "$test_project_name", "TestResults", "SpatialGDK.", "bEnableUnrealLoadBalancer=false$user_gdk_settings", $True)
```

Comment:
Why do we now need to add `.` here?

Reply:
Unreal runs tests by doing string find of the supplied string. So when we have a category called `SpatialGDK` and then also `SpatialGDKSlow`, specifying `SpatialGDK` to run tests would run both. But since tests have fully qualified names like `SpatialGDK.Core.SomeCoreTestName`, we can specify `SpatialGDK.` to only run the tests in the `SpatialGDK` category and exclude the `SpatialGDKSlow` tests.

Yes, unfortunate. I think the alternative would be to build some custom filtering, but seemed excessive just for this. Or we could rename `SpatialGDKSlow` into `SpatialSlow` or something.

Reply:
Ah, that makes sense, cool 👍